core/chains/evm/types/types_test.go

```diff
@@ -0,0 +1,37 @@
+package types_test
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/smartcontractkit/chainlink/core/chains/evm/types"
+	"github.com/smartcontractkit/chainlink/core/internal/cltest"
+	"github.com/smartcontractkit/chainlink/core/internal/testutils/pgtest"
+	"github.com/smartcontractkit/chainlink/core/utils"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
minor point: third part imports and internal ones should be swapped I believe :)

Reply:
Out of curiosity what is the reasoning for this?

Reply:
@samsondav just convention. It is really convenient to read imports if they are sorted in that way. Nothing super important but nice to have.